hypothesis-python/src/hypothesis/internal/reflection.py

```diff
@@ -22,8 +22,10 @@
 from tokenize import detect_encoding
 from types import ModuleType
 from typing import TYPE_CHECKING, Callable
+from unittest.mock import _patch as PatchType
```

Comment:
Unfortunate `unittest` doesn't expose this publicly, but I guess this has and will remain pretty stable.

Reply:
Yeah, our internals do (have to) depend on a lot of technically-private stuff.  Best mitigation is extensive testing, including on alpha versions of new Python releases - if something breaks we should find out far enough in advance to fix it before the stable release.